File: app/src/main/java/com/yenvth/soilDetectionApp/labeling/LabelingView.java
package com.yenvth.soilDetectionApp.labeling;

import com.yenvth.soilDetectionApp.models.LabelingModel;

import java.util.ArrayList;

public interface LabelingView {
    void onGetListLabelSuccess(ArrayList<LabelingModel> list);

    void addImageFirebaseSuccess(String url);

    void onAddLabelSuccess();

    void onDeleteLabelSuccess();
}

File: app/src/main/java/com/yenvth/soilDetectionApp/labeling/LabelingView.kt
package com.yenvth.soilDetectionApp.labeling

import com.yenvth.soilDetectionApp.models.LabelModel

interface LabelingView {
    fun onGetListLabelSuccess(list: List<LabelModel>)
    fun onAddLabelSuccess()
    fun onDeleteLabelSuccess()
    fun showLoading()
    fun hideLoading()
}